opm/simulators/linalg/WriteSystemMatrixHelper.hpp

```diff
@@ -22,6 +22,46 @@
 
 #include <dune/istl/matrixmarket.hh>
 
+namespace Dune
+{
+namespace MatrixMarketImpl
+{
+
+    template <typename Block, typename A>
```

Comment:
Might be my aura or DUNE version (2.8), but suddenly I am getting compile errors due to redefinitions here:
```
/home/mblatt/src/dune/opm/opm-simulator/opm/simulators/linalg/WriteSystemMatrixHelper.hpp: At global scope:
/home/mblatt/src/dune/opm/opm-simulator/opm/simulators/linalg/WriteSystemMatrixHelper.hpp:31:12: error: redefinition of ‘struct Dune::MatrixMarketImpl::mm_header_printer<Dune::BCRSMatrix<B, TA> >’
     struct mm_header_printer<BCRSMatrix<Block, A>>
            ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
In file included from /home/mblatt/src/dune/opm/dune-istl/dune/istl/owneroverlapcopy.hh:33:0,
                 from /home/mblatt/src/dune/opm/opm-grid/opm/grid/cpgrid/CpGridData.hpp:56,
                 from /home/mblatt/src/dune/opm/opm-grid/opm/grid/cpgrid/Iterators.hpp:142,
                 from /home/mblatt/src/dune/opm/opm-grid/opm/grid/cpgrid/Entity.hpp:338,
                 from /home/mblatt/src/dune/opm/opm-grid/opm/grid/CpGrid.hpp:67,
                 from /home/mblatt/src/dune/opm/opm-simulator/ebos/eclbasevanguard.hh:34,
                 from /home/mblatt/src/dune/opm/opm-simulator/ebos/eclcpgridvanguard.hh:30,
                 from /home/mblatt/src/dune/opm/opm-simulator/ebos/eclproblem.hh:52,
                 from /home/mblatt/src/dune/opm/opm-simulator/opm/simulators/flow/BlackoilModelEbos.hpp:27,
                 from /home/mblatt/src/dune/opm/opm-simulator/opm/simulators/flow/FlowMainEbos.hpp:28,
                 from /home/mblatt/src/dune/opm/opm-simulator/opm/simulators/flow/Main.hpp:52,
                 from /home/mblatt/src/dune/opm/opm-simulator/flow/flow_blackoil_dunecpr.cpp:23:
/home/mblatt/src/dune/opm/dune-istl/dune/istl/matrixmarket.hh:175:12: note: previous definition of ‘struct Dune::MatrixMarketImpl::mm_header_printer<Dune::BCRSMatrix<B, TA> >’
     struct mm_header_printer<BCRSMatrix<T,A> >
            ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```

Seems like this might not work for DUNE 2.7 (same method definition. Only 2.6 (e.g. Debian Buster) explicitly used FieldMatrix as the block.

Reply:
will fix...